packages/cli/src/server/web/server.rs

```diff
@@ -92,6 +99,9 @@ pub async fn setup_router(
         router
     };
 
+    // Setup websocket
+    router = router.route("/_dioxus/ws", get(ws_handler));
```

Comment:
Calling out this change -- related to the change on :65-66 -- as I'm not 100% sure how this will interact with the proxy configuration on :69-71. I know the order of some methods called on axum routers matter, but I'm not terribly familiar with the framework. I'm also not sure how to correctly test this interaction.

Reply:
I believe axum matches exact routes first. I tested that hot reloading still works if the base_path is `_dioxus` and the proxy ends in `_dioxus`. The proxy just adds new routes, not layers so I don't think this will effect behavior